pkg/trait/camel.go

```diff
@@ -238,8 +239,9 @@ func (t *camelTrait) computeConfigMaps(e *Environment) []ctrl.Object {
 		)
 	}
 
-	for i, s := range sources {
-		if s.ContentRef != "" || e.isEmbedded(s) {
+	i := 0
```

Comment:
A little nitpick out of curiosity: what is the root cause for this change of starting index value from 1 to 0 ?

Reply:
the change is not the index itself. The problem is that if we let the variable as implicit part of the loop, it increments also when we skip the source, ie, is embedded, kamelet, etc. But we only want to increment when it is really processed, reason why we need to make it explicit.

Reply:
Thanks, forget the "1" part, I was mixing with the test result change.